wagtail/admin/views/tags.py

```diff
@@ -19,7 +19,7 @@ def autocomplete(request, app_name=None, model_name=None):
 
     term = request.GET.get("term", None)
     if term:
-        tags = tag_model.objects.filter(name__istartswith=term).order_by("name")
+        tags = tag_model.objects.filter(name__istartswith=term).order_by("name")[:10]
```

Comment:
Thanks, could you add a test that would fail if the `[:10]` weren't there?

Reply:
Sure, will do